src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
```

Comment:
I will not count it as a mistake, because operating the actual `Color`  values instead of the `String` representation of it is actually better, but it is also recommended to not change the signature of the method, so this is up to you
And if you have a Color instance here, you might as well have a `Ball.color` field  be of type `Color`